### PR TITLE
Update to Edit Card Packages

### PR DESCRIPTION
**Pull request by AlbertYS**
Changes to EditCardInputData such that it uses card id number rather than card itself, now accepts editing of options with overloaded constructors EditCardInputBoundary changes as required

Updated implementations for:
EditCardInteractor
EditCardInputBoundary
EditCardController

### CODE REVIEW

**Review by derrickcho3715 — DISMISSED** [review]
This looks good. All the proper documentation is there and it has all the necessary files including tests. One thing to note is that the EditCardInteractor is missing a call to EditCardOutputBoundary. Apart from that it's good work.

**Review by jakan9 — APPROVED** [review]

**Review by jakan9 — APPROVED** [review]

File: src/main/java/editCard/EditCardInteractor.java [hunk 1]
```diff
@@ -1,18 +1,31 @@
 package editCard;/*
  * Author: Albert Yan
  * Date: October 31/22
- * The use case interactor for editing cards a
+ * The use case interactor for editing cards given an input boundary
  */
 
-public class EditCardInteractor {
```
Comment (aldob02) [inline src/main/java/editCard/EditCardInteractor.java:7]: The interactor needs to implement the input boundary.

**Review by aldob02 — CHANGES_REQUESTED** [review]

**Review by aldob02 — APPROVED** [review]
Looks good, only testing and javadocs are needed.

**Review by derrickcho3715 — APPROVED** [review]
This looks good. All the proper documentation is there and it has all the necessary files including tests. The issue with the interactor and output boundary was properly addressed.

**Pull request merged**